MOVING.md

```diff
@@ -1,7 +1,12 @@
 # Moving to Core SDK
 
 ## What's changed
-While the code has moved the generation has stayed the same, which means updating to the Core SDK of the language you choose should be similar to updating to a new version in the old SDK.
+
+### Go
+While the code has moved the generation has stayed the same, which means updating to the Core SDK should be similar to updating to a new version in the old SDK
+
+### Java
+Java has been completly replaced with a new generator from Kiota you can learn more [here](https://microsoft.github.io/kiota/). This means any code in the previous Java SDK will no longer work and is no longer supported.
```

Comment:
this sounds a little harsh 😄 , the generated code looks pretty much similar and offers the functionality.
I would phrase it in a more "supportive" way, "try it out and ask questions and open issues if you find anything unexpected" ... something along those lines.

Reply:
Ah okay, again my knowledge on the new SDK is low so my thoughts on the Java SDK are not fully there until I get to try it with Rama. I will reword it as you know more about the transition